tests/baselines/reference/parseBigInt.errors.txt

```diff
@@ -82,13 +82,13 @@ tests/cases/compiler/parseBigInt.ts(70,72): error TS2345: Argument of type '3' i
 !!! error TS1005: ',' expected.
     { const scientific = 1e2n; }
                             ~
-!!! error TS1005: ',' expected.
+!!! error TS1351: An identifier or keyword cannot immediately follow a numeric literal.

```

Comment:
This could be better if you want to send a follow-up PR to see if you have a length of `1` and the identifier is named `n`.